README.md

```diff
@@ -1,68 +1,37 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Series-Rush
 
-## Available Scripts
+Track all your series and movies in one place.
 
-In the project directory, you can run:
+## Usage
 
-### `npm start`
+### Get the code
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.<br>
-You will also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.<br>
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.<br>
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+`git clone https://github.com/knightburton/series-rush.git`
 
-### `npm run eject`
+### Install npm
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+`curl -L https://www.npmjs.com/install.sh | sh`
```

Comment:
I would rather suggest to use nvm to install npm
`https://github.com/nvm-sh/nvm`